.github/workflows/common_library_tests.yaml

```diff
@@ -164,33 +167,14 @@ jobs:
           # Flags found here https://github.com/k3d-io/k3d
           k3d-args: --k3s-arg --disable=metrics-server@server:*
 
-        # Install Kail to grab logs from tests, as there are cases ct-install fail to output logs
-      - name: Install Kail
-        run: |
-          export KAIL_VERSION=v0.16.1
-          wget https://github.com/boz/kail/releases/download/${KAIL_VERSION}/kail_${KAIL_VERSION}_linux_amd64.tar.gz
-          tar -xvzf kail_${KAIL_VERSION}_linux_amd64.tar.gz
-          chmod +x kail
-
       - name: Run chart-testing (install)
         run: |
+          # This is done to improve speed of the tests, so we can run tests in parallel
           # Move all ci values on a temp location (or skip if already moved from another matrix job)
           mv library/common-test/ci library/common-test/runtests || echo "Nothing to move"
 
           # Move one values.yaml to the correct location to run the test
           mv -f library/common-test/runtests/${{ matrix.values }} library/common-test/values.yaml
 
-          # Stat kail on the background to grab logs from tests
-          ./kail --ignore-ns kube-system >> /tmp/output.log &
-
           # Actually run the test
-          ct install  --config .github/ct-install-config/ct-install.yaml \
-                      --charts library/common-test \
-                      --debug || (echo -e "\n\n--===PODLOGS===--\n\n" && \
-                                  cat /tmp/output.log && \
-                                  rm -f /tmp/output.log && exit 1)
-
-          kill $!
-          echo -e "\n\n--===PODLOGS===--\n\n"
-          cat  /tmp/output.log
-          rm -f /tmp/output.log
```

Comment:
Initially was added because sometimes CI was faling to capture and display all logs.
hopping that the issue was coming from the specific action, added the above.
But this didn't make a difference. So removing it.